editor/debugger/script_editor_debugger.cpp

```diff
@@ -891,6 +950,41 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, uint64_t p_thread
 	}
 }
 
+HashMap<String, ScriptEditorDebugger::ParseMessageFunc> *ScriptEditorDebugger::parse_message_handlers = nullptr;
+
+void ScriptEditorDebugger::_init_parse_message_handlers() {
+	HashMap<String, ScriptEditorDebugger::ParseMessageFunc> *handlers = memnew((HashMap<String, ScriptEditorDebugger::ParseMessageFunc>));
```

Comment:
Isn't a HashMap default initialized? I don't recall us doing explicit `memnew` for HashMaps.

Reply:
We are using a lot of static `HashMap`s, so `memnew` is not necessary. Also, nothing is freeing it, so map will leak.

Reply:
Done!